applications/GeoMechanicsApplication/geo_mechanics_application_variables.h

```diff
@@ -0,0 +1,186 @@
+    KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS( GEO_MECHANICS_APPLICATION, LOCAL_RELATIVE_DISPLACEMENT_VECTOR )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, Matrix, PERMEABILITY_MATRIX )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, Matrix, LOCAL_PERMEABILITY_MATRIX )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, CRITICAL_DISPLACEMENT )
+    KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS( GEO_MECHANICS_APPLICATION, TOTAL_DISPLACEMENT )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE(GEO_MECHANICS_APPLICATION, bool, IS_CONVERGED)
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, Matrix, TOTAL_STRESS_TENSOR )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, Vector, TOTAL_STRESS_VECTOR )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, Matrix, CAUCHY_STRAIN_TENSOR )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, Vector, CAUCHY_STRAIN_VECTOR )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, STATE_VARIABLE )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, ARC_LENGTH_LAMBDA )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, ARC_LENGTH_RADIUS_FACTOR )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, TIME_UNIT_CONVERTER )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, LOCAL_EQUIVALENT_STRAIN )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, NONLOCAL_EQUIVALENT_STRAIN )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, JOINT_WIDTH )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, bool,   NODAL_SMOOTHING )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, Matrix, NODAL_CAUCHY_STRESS_TENSOR )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, NODAL_DAMAGE_VARIABLE )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, NODAL_JOINT_AREA )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, NODAL_JOINT_WIDTH )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, double, NODAL_JOINT_DAMAGE )
+
+    // Nodal load variables
+    KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS( STRUCTURAL_MECHANICS_APPLICATION, POINT_LOAD )
+    KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS( STRUCTURAL_MECHANICS_APPLICATION, LINE_LOAD )
+    KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS( STRUCTURAL_MECHANICS_APPLICATION, SURFACE_LOAD )
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, bool, RESET_DISPLACEMENTS )
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, bool, CONSIDER_GEOMETRIC_STIFFNESS)
+
+    KRATOS_DEFINE_APPLICATION_VARIABLE( GEO_MECHANICS_APPLICATION, bool, USE_CONSISTENT_MASS_MATRIX)
```

Comment:
this was removed in structural

=> now we use `COMPUTE_LUMPED_MASS_MATRIX` from the Core

Reply:
This part will definitely be removed. All the elements that I borrowed from Structural Mechanics will be removed as soon as the reseting displacement issue is solved in that application.